src/content/docs/devtools/Docker/docker-swarm-case-study.md

```diff
@@ -0,0 +1,206 @@
+| Attribute  | Details                                                      |
+|------------|--------------------------------------------------------------|
+| Title      | Docker swarm case study                                            |
+| Author     | Malene Ivy Olsen                                   |
+| Tags       | docker, container, containerization, swarm, container orchestration, kubernetes | <br/>
+<br/>
+
+# Docker Swarm - container orchestration tool
+[Acronyms used in the text](#acronyms-used)
+## Introduction
+This case study aims to compare Docker and Kubernetes, the two key technologies in containerization and orchestration. I'll begin by introducing Docker, an essential tool for containerization, to set the groundwork for understanding container orchestration tools. <br/>
+Then I'll dive into comparison of Docker Swarm and Kubernetes to help students/beginners understand these important tools in modern software development. 
+
+## Containerization
+A container works as a self-contained environment that holds all of the necessary components for running an application. It provides an isolated environment for running applications and services. <br/>
+Containerization enables teams to package applications, dependencies and configurations into container images. This simplifies the development of microservices, which are small, independent software components that focus on specific tasks within a larger application. <br/>
+
+### Container Key Milestones
+![Container key milestones after 2013](../../../../assets/devtools/docker/docker-history-timeline.jpg)
+In 1979, the concept of containers first emerged. This was 34 years before Docker surfaced in 2013. <br/>
```

Comment:
There are 43 instances of <br> to be removed, I will no longer be commenting on them.